corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -456,8 +456,8 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             // only here to maintain backward compatibility and can be removed
             // once web apps fully transition using keys to convey select prompt selection.
             this.selectValuesByKeys = false;
-            this.dynamicSearchEnabled = toggles.toggleEnabled('DYNAMICALLY_UPDATE_SEARCH_RESULTS') &&
-                this.options.sidebarEnabled;
+            this.dynamicSearchEnabled = options.disableDynamicSearch ? false :
+                (toggles.toggleEnabled('DYNAMICALLY_UPDATE_SEARCH_RESULTS') && this.options.sidebarEnabled);
```

Comment:
nit: you could simplify this a bit by moving `dynamicSearchEnabled` to the initialization options, then set it to `false` for the initial page load and `toggleEnabled(...)` for the other ones in `controller.js`.  (Am I correct in interpreting that `QueryListView` is reinitialized each update?)

Reply:
^ comment non-blocking

Reply:
Ah I like this-- yes `QueryListView` is initialized each time. Given the time sensitivity of this feature, I'll merge for now and make a note to followup-- thanks!